docs/installation/helm.md

```diff
@@ -83,29 +165,34 @@ This command initializes Minikube, and it may take a few minutes as it downloads
 🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
 
 ```
+
 To verify the status of your Minikube cluster, run:
+
+```bash
+minikube status
 ```
-$ minikube status
+
+You should see a status indicating that the cluster is running.
+
+```bash
```

Comment:
After this, also mention a step to do a `kubectl get nodes` to make sure that the minikube node is up and running